### PR TITLE
[3384][ADD] stock_picking_product_type

### PR DESCRIPTION
**Pull request by rinaldifirdaus**
[3384](https://www.quartile.co/web?db=quartile&token=dMRr78CMkEX6gyy9nTiv#id=3384&action=771&model=project.task&view_type=form&menu_id=505)

In current code, I am trying to hide the product type on picking code other than "incoming".
But the result is, the field is still shown only value is hidden.
I will discuss with client whether the product type is exclusively want to be shown only in Reception or it's ok to be displayed on the other picking code.

### CODE REVIEW

File: stock_move_product_type/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+This module displays product type in lines of stock move.
```
Comment (yostashiro) [inline stock_move_product_type/readme/DESCRIPTION.rst]:
```suggestion
This module displays product type in stock moves in the picking form.
```

**Review by yostashiro — COMMENTED** [review]
Can we change the module name to stock_picking_product_type?

**Comment by yostashiro** [comment]
> I will discuss with client whether the product type is exclusively want to be shown only in Reception or it's ok to be displayed on the other picking code.

I would personally go for the simpler option of showing the field in all pickings first when I am not convinced with the reason to do otherwise.

**Comment by rinaldifirdaus** [comment]
@yostashiro well noted. I will let the field and its value to be shown in all picking. and will follow up if there's any feedback or additional request from client.  Thank you

**Review by yostashiro — APPROVED** [review]

**Pull request merged**